deps.edn

```diff
@@ -1,18 +1,25 @@
 {:paths   ["src" "classes"]
 
  :deps
-          {org.clojure/clojure               {:mvn/version "1.10.1"}
-           clj-time                          {:mvn/version "0.14.2"}
-           javax.servlet/servlet-api         {:mvn/version "2.5"}
-           org.apache.activemq/activemq-core {:mvn/version "5.7.0"}
-           org.slf4j/slf4j-simple            {:mvn/version "1.7.30"}
-           amperity/vault-clj                {:mvn/version "0.5.1"}
-           org.apache.commons/commons-email  {:mvn/version "1.5"}
-           com.taoensso/timbre               {:mvn/version "4.10.0"}}
+          {org.clojure/clojure                   {:mvn/version "1.10.1"}
+           org.clojure/data.json                 {:mvn/version "1.0.0"}
+           clj-time                              {:mvn/version "0.14.2"}
+           cljfmt                                {:mvn/version "0.6.7"}
```

Comment:
Is `cljfmt` a runtime dependence or just required for development? Can it be made a dependence of the `lint` and `format` aliases?

Reply:
Good catch! Lemme see if we could move it to the aliases!!